### PR TITLE
Feat(Query): added "Kubelet Streaming Connection Timeout Disabled" for Kubernetes

### PR DESCRIPTION
**Pull request by cxAndreFelicidade**
**Proposed Changes**
- Query: Streaming Connection Timeout Disabled for Kubernetes Yaml

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/k8s/streaming_connection_timeout_disabled/query.rego [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	common_lib.inArray(container.command, command)
+	k8sLib.hasFlag(container, "--streaming-connection-idle-timeout=0")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("metadata.name={{%s}}.%s.%s.name={{%s}}.command", [metadata.name, specInfo.path, types[x], container.name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "--streaming-connection-idle-timeout flag is not 0 or not set",
+		"keyActualValue": "--streaming-connection-idle-timeout flag is 0",
+		"searchLine": common_lib.build_search_line(["metadata", "name"], []),
```
Comment (rafaela-soares) [inline assets/queries/k8s/streaming_connection_timeout_disabled/query.rego:23]:
```suggestion
        "searchLine": common_lib.build_search_line(split(specInfo.path, "."), [types[x], j, "command"])
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: assets/queries/k8s/streaming_connection_timeout_disabled/metadata.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "ed89b97d-04e9-4fd4-919f-ee5b27e555e9",
+  "queryName": "Streaming Connection Timeout Disabled",
```
Comment (rafaela-soares) [inline assets/queries/k8s/streaming_connection_timeout_disabled/metadata.json:3]:
```suggestion
       "queryName": "Kubelet Streaming Connection Timeout Disabled",
```

Please, also update the query folder name and the `positive_expected_result.json`

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: assets/queries/k8s/kubelet_streaming_connection_timeout_disabled/query.rego [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+	command := "kubelet"
+
+	common_lib.inArray(container.command, command)
+	k8sLib.hasFlag(container, "--streaming-connection-idle-timeout=0")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("metadata.name={{%s}}.%s.%s.name={{%s}}.command", [metadata.name, specInfo.path, types[x], container.name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "--streaming-connection-idle-timeout flag is not 0 or not set",
```
Comment (rafaela-soares) [inline assets/queries/k8s/kubelet_streaming_connection_timeout_disabled/query.rego:21]:
```suggestion
        "keyExpectedValue": "--streaming-connection-idle-timeout flag is not 0",
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: assets/queries/k8s/kubelet_streaming_connection_timeout_disabled/test/positive1.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: command-demo
+  labels:
+    purpose: demonstrate-command
+spec:
+  containers:
+    - name: command-demo-container
+      image: gcr.io/google_containers/kube-apiserver-amd64:v1.6.0
```
Comment (joaorufi) [inline assets/queries/k8s/kubelet_streaming_connection_timeout_disabled/test/positive1.yaml:10]: Use `kubelet` docker image
```suggestion
      image: joaodanielrufino/kubelet
```

File: assets/queries/k8s/kubelet_streaming_connection_timeout_disabled/test/negative1.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: command-demo
+  labels:
+    purpose: demonstrate-command
+spec:
+  containers:
+    - name: command-demo-container
+      image: gcr.io/google_containers/kube-apiserver-amd64:v1.6.0
```
Comment (joaorufi) [inline assets/queries/k8s/kubelet_streaming_connection_timeout_disabled/test/negative1.yaml:10]:
```suggestion
      image: joaodanielrufino/kubelet
```

File: assets/queries/k8s/kubelet_streaming_connection_timeout_disabled/query.rego [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+	command := "kubelet"
+
+	common_lib.inArray(container.command, command)
+	k8sLib.hasFlag(container, "--streaming-connection-idle-timeout=0")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("metadata.name={{%s}}.%s.%s.name={{%s}}.command", [metadata.name, specInfo.path, types[x], container.name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "--streaming-connection-idle-timeout flag is not 0",
```
Comment (joaorufi) [inline assets/queries/k8s/kubelet_streaming_connection_timeout_disabled/query.rego:21]:
```suggestion
		"keyExpectedValue": "--streaming-connection-idle-timeout flag not to be 0",
```

File: assets/queries/k8s/kubelet_streaming_connection_timeout_disabled/query.rego [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+CxPolicy[result] {
+	resource := input.document[i]
+    resource.kind == "KubeletConfiguration"
+    resource.streamingConnectionIdleTimeout == "0s"
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": "kind={{KubeletConfiguration}}.streamingConnectionIdleTimeout",
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "streamingConnectionIdleTimeout is not 0s",
```
Comment (joaorufi) [inline assets/queries/k8s/kubelet_streaming_connection_timeout_disabled/query.rego:36]:
```suggestion
		"keyExpectedValue": "streamingConnectionIdleTimeout not to be 0s",
```

**Review by joaorufi — CHANGES_REQUESTED** [review]

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

**Review by rafaela-soares — DISMISSED** [review]

**Review by joaorufi — APPROVED** [review]

**Pull request merged**